LimeBean/BeanApi.cs

```diff
@@ -573,6 +573,10 @@ public partial class BeanApi : IDisposable, IBeanCrud, IBeanFinder, IDatabaseAcc
             set { Storage.ConvertEmptyStringToNull = value; }
         }
 
+        /// <summary>
+        /// Gets or Sets whether integers are detected and extracted if in string type variables, when storing to the database.
+        /// Query paramaterisation and the database still handle storage types
+        /// </summary>
         public bool RecognizeIntegers {
```

Comment:
It's useful in fluid mode. For example, if the column "n" is new, then it will be created with a low rank (8-bit integer, maybe) rather than string:
```
bean["n"] = "123"
api.Store(bean);
```